addons/mail/wizard/mail_activity_schedule.py

```diff
@@ -150,21 +150,26 @@ def _compute_plan_id(self):
     def _compute_plan_date(self):
         self.plan_date = False
 
+    def _get_summary_lines(self, templates):
+        self.ensure_one()
+        summaries = []
+        for template in templates:
+            summary_line = template.activity_type_id.name
+            if template.summary:
+                summary_line += f": {template.summary}"
+            # We don't display deadlines when the user doesn't specify a plan_date
+            if self.plan_date:
+                summary_line += f" ({format_date(self.env, template._get_date_deadline(self.plan_date))})"
+            summaries.append('<li>%s</li>' % summary_line)
+        return ''.join('<ul>%(template)s</ul>' % {'template': "".join(summaries)}) if summaries else ''
```

Comment:
Actually why not return Markup here directly? Seems to make more sense, as we know it's safe here. And avoids having to use Markup around the call to this, probably every time

Reply:
When I did that, I had some problems with html tags ; they was considered as a part of the string :sweat:

Reply:
Not sure where your problematic tags come from but maybe you can make them Markup there? Or maybe you're removing the markup and then re-applying it.

In case you're not familiary, markup is actually really simple:
- It's a wrapper around a string, meaning it's a string for all intents and purposes except the following
- If you format a regular string into a Markup string, it will clean up the string (escaping HTML nodes so they're only visual)  ->   `Markup('{test}').format(test="<html></html>") == Markup('\<html\>\<\/html\>')`
- If you format a Markup string into a Markup, it formats as if they were two strings except the result is a markup string -> `Markup('{test}').format(test=Markup('<html></html>')) == Markup('<html></html>')` 
- If you format a Markup string into a regular string, then you lose the "markup" element of course -> `"".join([Markup("<html></html>")]) == "<html></html>`

So the goal of `Markup` is that you use it around html tags, so they don't get mangled, and you insert everything else that isn't supposed to be HTML into it via formatting so it doesn't look weird in the UI (or worse).

For example above in `'<ul>%(responsible)s %(block)s</ul>'` you should ideally use `Markup('<ul>%(responsible)s %(block)s</ul>') % (responsible=..., block=...)` so it's clear the only Markup we're saying should be kept is `<ul>` and whatever markup may already be in the arguments. Then `block` should also return `Markup` so its html nodes are not escaped, etc... It's better if it's systematic accross function calls rather than having to wonder whether making the result of the whole call stack `Markup` all at once is actually ok or not